db/schema.rb

```diff
@@ -0,0 +1,61 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `rails
+# db:schema:load`. When creating a new database, `rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 2020_04_03_011120) do
+
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "activities", force: :cascade do |t|
+    t.string "name"
+    t.time "est_time"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+  end
+
+  create_table "categories", force: :cascade do |t|
+    t.string "name"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+  end
+
+  create_table "category_activities", force: :cascade do |t|
+    t.bigint "category_id", null: false
+    t.bigint "activity_id", null: false
+    t.index ["activity_id"], name: "index_category_activities_on_activity_id"
+    t.index ["category_id"], name: "index_category_activities_on_category_id"
+  end
+
+  create_table "user_activities", force: :cascade do |t|
+    t.bigint "user_id", null: false
+    t.bigint "category_activity_id", null: false
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.index ["category_activity_id"], name: "index_user_activities_on_category_activity_id"
+    t.index ["user_id"], name: "index_user_activities_on_user_id"
+  end
+
+  create_table "users", force: :cascade do |t|
+    t.string "first_name"
+    t.string "last_name"
+    t.string "email"
+    t.string "google_token"
+    t.integer "self_care_time"
+    t.datetime "created_at", precision: 6, null: false
```

Comment:
Good to see the user structure, i'll try mirror accordingly in my initial mock data.